apps/studio/components/interfaces/Settings/Logs/LogsQueryPanel.tsx

```diff
@@ -99,25 +100,28 @@ const LogsQueryPanel = ({
               </DropdownMenuContent>
             </DropdownMenu>
 
-            <DropdownMenu>
-              <DropdownMenuTrigger asChild>
-                <Button type="default" iconRight={<IconChevronDown />}>
-                  Templates
-                </Button>
-              </DropdownMenuTrigger>
-              <DropdownMenuContent side="bottom" align="start">
-                {templates
-                  .sort((a, b) => a.label!.localeCompare(b.label!))
-                  .map((template) => (
-                    <DropdownMenuItem
-                      key={template.label}
-                      onClick={() => onSelectTemplate(template)}
-                    >
-                      <p>{template.label}</p>
-                    </DropdownMenuItem>
-                  ))}
-              </DropdownMenuContent>
-            </DropdownMenu>
+            {IS_PLATFORM && (
```

Comment:
might be good to have a `<PlatformOnly>` component for this kind of situations. just a thought